ParisBNF/abb/BNFabb106.xml

```diff
@@ -0,0 +1,234 @@
+                        </origin>
+                        <acquisition>The manuscript entered <placeName ref="INS0303BNF"/> in
+                            <date>1902</date>.</acquisition>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:ContiRossini1914Notice"/>
+                                            <citedRange unit="page">120</citedRange>
+                                            <citedRange unit="number">88</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Chaine1912Catalogue"/>
+                                            <citedRange unit="page">66</citedRange>
+                                            <citedRange unit="number">106</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:Abbadie1859Cat"/>
+                                            <citedRange unit="page">118-119</citedRange>
+                                            <citedRange unit="number">106</citedRange>
+                                        </bibl>
+                                        <bibl>
+                                            <ptr target="bm:BmWebsite"/>
+                                        </bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Chants"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Liturgy"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="NV" when="2024-09-09">I have complemented Conti Rossini's description with information provided by d'Abbadie and, based on the available images, described quire structure, ruling and stamp</change>
```

Comment:
do not use "I have" as NV already stands for "Nafisa Valieva". The text you created says
`Nafisa Valieva I have complemented
`it should be
`Nafisa Valieva complemented`